content.yaml

```diff
@@ -1015,3 +1015,127 @@ long_summary_md: |
+    title: 'Devops Desasters 3.1'
+  - start: '01:09:00'
+    title: 'This Is Not A Game'
+  - start: '01:09:55'
+    title: 'C3-PR'
+  - start: '01:10:16'
+    title: 'Fuzzing the Phone in the iPhone'
+  - start: '01:10:50'
+    title: 'Haecksen Goodies'
+  - start: '01:13:40'
+    title: 'C3WOC - Waffeln rund um die Uhr'
+  - start: '01:14:27'
+    title: 'Die Hacker-Genossenschaft: WTF e. G. (i. G.)'
+  - start: '01:14:45'
+    title: 'Uwe Lübbermann und Premium Cola'
+  - start: '01:15:45'
+    title: 'WTF e. G. (i. G.)'
+  - start: '01:16:10'
+    title: 'Engelgewerkschaft'
+  - start: '01:17:08'
+    title: 'Ausblick 2021'
+  - start: '01:18:16'
+    title: 'Outro'
+  - start: '01:19:57'
+    title: 'Fairy Bot Orchestra - I'm a Teapot'
+    href: 'https://fairybotorchestra.bandcamp.com/track/im-a-teapot-the-418-song'
+long_summary_md: |
+  **Shownotes:**
+
+  * [Golem: WhatsApp stellt Nutzern ein Ultimatum](https://www.golem.de/news/facebook-whatsapp-stellt-nutzern-ein-ultimatum-2101-153215.html)
+  * [Apple: Privacy Labels](https://developer.apple.com/news/?id=3wann9gh)
+  * [Forbes: Privacy Labels der Messenger im Vergleich](https://www.forbes.com/sites/zakdoffman/2021/01/03/whatsapp-beaten-by-apples-new-imessage-update-for-iphone-users/)
+  * WhatsApp Alternativen: [Signal](https://signal.org), [Threema](https://threema.ch/de), [Element](https://element.io/)/[Matrix](https://matrix.org/), 
+    [Rocketchat](https://rocket.chat/), [Telegram](https://telegram.org/)
+  * [Heise: Signal Messenger - Ansturm neuer Nutzer überlastet Signals Anmeldesystem](https://www.heise.de/news/Signal-Messenger-Ansturm-neuer-Nutzer-ueberlastet-Signals-Anmeldesystem-5018215.html)
+  * [Threema: jetzt OpenSource](https://threema.ch/de/open-source), [Threema: Code audits](https://threema.ch/es/faq/code_audit), [Threema: Multi-Device](https://threema.ch/de/blog/posts/md-architectural-overview-intro-de)
+  * [F-Droid, eine Google Play Store Alternative](https://f-droid.org/)
+  * [Chaos im Radio vom 9.11.2020 mit kurzem Teil über F-Droid](https://radio.ccc-p.org/#nt-2020-11-09)
+  * [Linksammlung](https://crypto.stackexchange.com/questions/31418/signal-vs-telegram-in-terms-of-protocols) zum Vergleich der kryptografischen Protokolle von Telegram und Signal
+  * Schmuddel-App Telegram: [Funk: die schlimmste App der Welt](https://www.funk.net/channel/walulis-1031/die-schlimmste-app-der-welt-walulis-1693617)
+  * [@HonkHase auf Twitter zum Netzfrequenz-Abfall](https://twitter.com/HonkHase/status/1347587723599732740)
+  * [@Schranke5 auf Twitter mit Frequenz-Diagram](https://twitter.com/Schranke5/status/1347617413576216580/photo/1)
+  * [Griradar: Diagramm](https://gridradar.net/unterfrequenz-49.75-Hz-Januar_2021)
+  * [Wikipedia: Unterfrequenz](https://de.wikipedia.org/wiki/Unterfrequenz)
+  * Kleinste Frequenz war bei 49,738 Hz .. bei 49,8 Hz wird Regelleistung eingesetzt - Speicherpumpkraftwerke schalten Pumpen ab und gehen in Turbinenbetrieb.
```

Comment:
Is this an errata? Shouldn't it go up to before the shownotes?

Reply:
Nein, das hab ich aus dem Pad übernommen.

Reply:
Hm, das ist wohl aus den Notizen mit reingerutscht, fliegt wieder raus